modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -65,6 +66,18 @@ def validate_json_schema_for_body
     AppealsApi::FormSchemas.new.validate!(self.class::FORM_NUMBER, @json_body)
   end
 
+  def validate_json_schema_for_pdf_fit
+    status, error = AppealsApi::HigherLevelReviews::PdfFormatValidation.new(
```

Comment:
When I initially saw this class name `PdfFormatValidation` I thought of validations of the pdf itself (dimensions, file size, etc) and was somewhat confused by it... Maybe it's just me, but something like `PdfFormFieldValidation` (or something completely different) would be more clear about what is being checked...